server/src/db/index.ts

```diff
@@ -0,0 +1,48 @@
+import mongoose, {Schema} from 'mongoose';
+
+// Recruiter accounts
+let UserSchema : Schema = new Schema({
+  name: String,
+  email: String,
+  password: String
```

Comment:
`passwordHash`? We can figure out which hash later

Reply:
Oh man I really wanted to store plaintext passwords

Reply:
Salting and hashing w/ some middleware will be pretty straightforward https://www.mongodb.com/blog/post/password-authentication-with-mongoose-part-1